hil/ext/switches/mock.py

```diff
@@ -56,6 +56,10 @@ def validate(kwargs):
             'password': basestring,
         }).validate(kwargs)
 
+    @staticmethod
+    def validate_port_name(port):
```

Comment:
Suggestion: put a comment here being explicit about not having specific requirements on the form of the port name.